specification/terraform/Microsoft.AzureTerraform.Management/models.tsp

```diff
@@ -48,7 +50,7 @@ model BaseExportModel {
 }
 
 @doc("Export parameter for resources queried by ARG (Azure Resource Graph)")
-model ExportQuery extends BaseExportModel {
+model ExportQuery extends CommonExportProperties {
```

Comment:
how about this model name `ExportQuery`?

Reply:
I prefer to keep the name `ExportQuery` because it's already present in the CLI. Refer to this previous discussion: https://github.com/Azure/azure-sdk-for-net/pull/46301#discussion_r1811724646.